src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { User } from '../types/User';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const { name, email } = user;
+
+  return (
+    <a className="UserInfo" href={`mailto:${email}`}>
```

Comment:
Consider adding a `title` attribute to the anchor element to improve accessibility. Example: `title="Email ${name}"` 📧